docs/sources/datasources/testdata/_index.md

```diff
@@ -114,3 +114,19 @@ That makes it much easier for the developers to replicate and solve your issue.
 [panels-visualizations]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations"
 [panels-visualizations]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations"
 {{% /docs/reference %}}
+
+## Use a custom version of TestData
+
+{{% admonition type="note" %}}
+This feature is experimental and requires Grafana version 10.3.0 as a minimum.
+{{% /admonition %}}
+
+If you want to use a version of TestData different from the one shipped with Grafana, you can do so by following these steps:
+
+1. Enable the [feature toggle](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/) `externalCorePlugins`.
+1. Set the configuration field `plugin.grafana-testdata-datasource.as_external` to `true`.
```

Comment:
I wonder is this explicit enough for people to know what to put in the `ini`? Just not seen it before described this way

Reply:
fair point :+1: